packages/kotti-ui/source/kotti-popover/KtPopover.vue

```diff
@@ -19,13 +26,27 @@
 import { createPopper } from '@popperjs/core'
 import { mixin as clickaway } from 'vue-clickaway'
 
+import KtPopoverIconTextItem from './KtPopoverIconTextItem.vue'
+
+const optionIsValid = (option) =>
+	(!option.isDisabled || typeof option.isDisabled === 'boolean') &&
+	option.onClick &&
```

Comment:
same as the `isDisabled` you don't want to force the user to pass an `onClick` 
if the user doesn't want the onClick option, then his item is not clickable, easy

my suggestion would be similiar then
either typeof `onClick` is `undefined` (so not passed)
or is a `function`

Reply:
@carsoli why would we ever want an item that’s not disabled but not clickable? I’m sorta fine with not necessarily having an `onClick` when it’s disabled though